src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -364,7 +364,20 @@ private BoundExpression CreateTupleLiteralConversion(CSharpSyntaxNode syntax, Bo
             {
                 var argument = arguments[i];
                 var destType = targetElementTypes[i];
-                convertedArguments.Add(CreateConversion(argument, destType, diagnostics));
+
+                HashSet<DiagnosticInfo> useSiteDiagnostics = null;
+                Conversion elementConversion;
+                if (isCast)
+                {
+                    elementConversion = this.Conversions.ClassifyConversionForCast(argument, destType, ref useSiteDiagnostics);
```

Comment:
Yes. That is the next change that I mentioned in the description. We might not need to be classifying anything at all when lowering conversions.